### ISSUE TITLE
No album cover displayed when playing music

### ISSUE THREAD

**Issue by MartijnKaijser** [freezy/android-xbmcremote#55]
v0.9.0
(other client does display it)

**Comment by MartijnKaijser** [comment]
Make that in general no artwork is shown like in library too

**Comment by tthomas48** [comment]
Are you running Frodo or Eden (or something else). Also if you're able to hook up logcat you should see a FileNotFoundException with an URL. If you could pull me one of those URLs that would be helpful. You should be seeing this sort of thing with Frodo. 
![Screenshot_2012-12-07-14-55-58.png](https://f.cloud.github.com/assets/168526/2525/962bef8a-422f-11e2-836b-f94f7e461dfa.png)

**Comment by MartijnKaijser** [comment]
Can't give logcat
Frodo openelec built from latest master. The version from tombriden does show artwork

**Comment by tthomas48** [comment]
I just got OpenElec running in a VM. To get images to work I had to enable 'Allow control of XBMC via HTTP'. Which is under Settings->Services->Webserver.

Then I had to change the HTTP API port in the XBMC android remote to port 80, since OpenElec binds to port 80 by default.

There's currently no way to get album art directly through the JSON API that I know of, so we have to keep that HTTP port open.
